src/keys.rs

```diff
@@ -17,6 +17,26 @@ pub fn keyval_to_unicode(keyval: u32) -> Option<char> {
     unsafe { ::std::char::from_u32(gdk_sys::gdk_keyval_to_unicode(keyval)).filter(|x| *x != '\0') }
 }
 
+pub fn keyval_is_lower(keyval: u32) -> bool {
```

Comment:
Do we really need these functions? Having them implemented on `Key` isn't enough? (Same question for the existing ones.)

Reply:
yah, I just copied the style of the existing ones

Reply:
Maybe we should just remove them... What do you think @sdroege ?

Reply:
I don't see the point in them either :)

Reply:
ok I'll do it